portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java

```diff
@@ -1029,8 +1027,6 @@ public PortalRenderContext startPageContext(String siteType, String title, Strin
 
         rcontext.put("notificationsPushEnabled", notificationsPushEnabled);
```

Comment:
I think this one is still left

Plus a couple properties:
- portal.title.shortdescription.show
- portal.notifications.push.enabled

Reply:
push.enabled needs to stay because the user messaging service uses it.

Reply:
True, I thought it wasn't used anywhere else